ac_posystem/src/pages/Payment.js

```diff
@@ -1,120 +1,166 @@
-import React from 'react'
+import React from "react";
 import { Link } from "react-router-dom";
+import axios from "axios";
 
+var total = 0 
+var totalsum = 0
+var itemss = []
 class Payment extends React.Component {
-    render() {
-        return (
-            <div>
-                <div>
-
-                    <div class="wrapper">
-                        <div class=" a">
-                            <div class="page">
-                                <section class="shopping-cart dark" style={{margin:100}}>
-                                    {/* <div class="container">
-                                        <div class="block-heading">
-                                        </div> */}
-                                        
-                                            <div class="summary">
-                                                <h3>Summary</h3>
-                                                <div class="summary-item"><span class="text">Subtotal</span><span class="price">$360</span>
-                                                </div>
-                                                <div class="summary-item"><span class="text">Discount</span><span class="price">$10</span></div>
-                                                <div class="summary-item"><span class="text">Shipping</span><span class="price">$5</span></div>
-                                                <div class="summary-item"><span class="text">Total</span><span class="price">$355</span></div>
-                                                <button type="button" class="btn btn-primary btn-lg btn-block">Continue Shopping</button>
-                                            </div>
-                                        
-                                    {/* </div> */}
-                                </section>
-                            </div>
-
+  //call backend when payment button is clicked and pass the sessionStorage data to backend
+  
+  sendData() {
+    // POST request using fetch inside useEffect React hook
+    itemss = JSON.parse(sessionStorage.getItem("items"));
+    itemss.forEach((item) => {
+      item.sc_image = "";
+    });
+    axios
+      .post("http://localhost:8803/updateInventory", itemss)
+      .then((res) => {
+        console.log(res);
+        console.log(res.data);
```

Comment:
Code looks understandable but try to avoid printing response data in the console